hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocation.java

```diff
@@ -117,7 +119,11 @@ public ClientMessage getClientMessage() {
 
     public ClientInvocationFuture invoke() {
         assert (clientMessage != null);
-        clientMessage.setCorrelationId(callIdSequence.next());
+        try {
```

Comment:
Future note; would be nice of the call id sequence would immediately throw the right exception. Now there is catching noise.